client/Pages/EventTypeDetails/View.elm

```diff
@@ -734,7 +734,13 @@ consumingQueriesPanel queryStore =
             List.length sqlQueriesList
 
         countStr =
-            pluralCount count "consuming nakadi-sql queries"
+            if count == 1 then
+                String.fromInt count
+                    ++ " consuming nakadi-sql query"
```

Comment:
Can be just `consuming nakadi-sql query`. It is clear from the text that it is only one.

Reply:
As a header `1 consuming nakadi-sql query` sounds better.
When we say `The` it kind of gives a special meaning for the query; like even if there is more queries in the future, this one will somehow be special.